bait-files/src/test/java/com/google/sps/LoginServletTest.java

```diff
@@ -43,7 +43,7 @@ public void tearDown() {
 
   // Add a user and ensure there is a new entry in database
   @Test
-  public void doPost_addsUserSuccessfully_returnsUserInDatabase() throws IOException, ServletException {
+  public void doPost_forNewUser_returnsUserInDatabase() throws IOException, ServletException {
```

Comment:
One more naming nitpick =P
...Looking at the final version of this test, I see that we are asserting/verifying two things:
1)  The user now exists in the database.
2)  A redirect happens.

So I'd change the last part of this test name to reflect that: _createsUserAndRedirects.
So: doPost_forNewUser_createsUserAndRedirects

Reply:
renamed